packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1693,7 +1693,7 @@ export class ContainerRuntime
 			groupedBatchingEnabled: this.groupedBatchingEnabled,
 		});
 
-		ReportOpPerfTelemetry(this.clientId, this.deltaManager, this.logger);
+		ReportOpPerfTelemetry(() => this.clientId, this.deltaManager, this.logger); // @note OpPerfTelemetry instatiation (original)
```

Comment:
Not clear why to wrap it again - you can pass this._getClientId directly

Reply:
Great point, I'll do that. And oops, noticed I left a comment I use for local tooling, will clean that up.